src/models/cvc.ts

```diff
@@ -48,14 +48,17 @@ const cvcDataStoreSchemaFields: Record<keyof Center, any> = {
 
 export interface CenterDetails extends Center, Document {}
 
-const cvcSchema: Schema<CenterDetails> = new Schema(cvcDataStoreSchemaFields);
+const cvcSchema: Schema<CenterDetails> = new Schema(cvcDataStoreSchemaFields, {
+  timestamps: {createdAt: 'created_at', updatedAt: 'updated_at'},
+});
 cvcSchema.index(
   {state_id: 1, district_id: 1, 'cowin.center_id': 1},
   {unique: true}
 );
 cvcSchema.index({district_id: 1}, {background: true});
 cvcSchema.index({'cowin.district_name': 1}, {background: true});
 cvcSchema.index({'cowin.pincode': 1}, {background: true});
+cvcSchema.index({updated_at: -1}, {background: true});
```

Comment:
I think you might need to add this key to the other indexes for it to work the way you intend it to for filtering